api/app/signals/apps/gisib/rest_framework/filters/feature_collection.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+from django.contrib.gis.geos import Polygon
+from django.db.models import Q
+from django_filters.rest_framework import FilterSet, filters
+
+
+class FeatureCollectionFilterSet(FilterSet):
+    id = filters.NumberFilter(field_name='gisib_id')
+
+    bbox = filters.CharFilter()  # min_lon, min_lat, max_lon, max_lat
```

Comment:
Nice, make sure the frontend devs know this as well. As far as I known Leaflet works with (lat, lon) coordinates.